### PR TITLE
Dataflow project not shown in New Menu

### PR DESCRIPTION
**Pull request by lak-proddev**
**JEE perspective**

image

**Java Perspective**

image

**Web Perspective**

image

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.dataflow.ui/plugin.xml [hunk 1]
```diff
@@ -294,4 +294,16 @@
     <extension point="org.eclipse.help.contexts">
        <contexts file="helpContexts.xml" />
     </extension>
+	<extension point="org.eclipse.ui.perspectiveExtensions">
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.ui/plugin.xml]: Indent looks off. Are you using tabs? We try to stick to spaces.

Comment (lak-proddev) [reply plugins/com.google.cloud.tools.eclipse.dataflow.ui/plugin.xml]: Yes,  done format with default settings. Updated to two spaces

**Comment by lak-proddev** [comment]
> Thanks. I wonder if there's any way control the order in which menu items appear. Ideally we'd like App Engine to come before Dataflow.

I didn't find a  solution to order menus. that's why I sent SS to show. :)

**Comment by lak-proddev** [comment]
> Thanks. I wonder if there's any way control the order in which menu items appear. Ideally we'd like App Engine to come before Dataflow.

Will push another change if i find any solution for ordering.

**Pull request merged**